openwhisk/index.md

```diff
@@ -121,7 +162,7 @@ You can find additional information about {{site.data.keyword.openwhisk_short}}
 * [Entity names](./openwhisk_reference.html#openwhisk_entities)
 * [Action semantics](./openwhisk_reference.html#openwhisk_semantics)
 * [Limits](./openwhisk_reference.html#openwhisk_syslimits)
-* [REST API](https://new-console.{DomainName}/apidocs/98)
+* [REST API](./openwhisk_reference.html#rest-api)
```

Comment:
@sefuhrer what do you think? Should we link directly to API rest UI or to the reference.html?
I will let you made the call.